core.js

```diff
@@ -56,6 +56,19 @@ function _check(buffer, headers, options) {
 	return true;
 }
 
+async function _checkSequence(sequence, tokenizer, options) {
+	options = {
+		position: 0,
+		...options
+	};
+
+	const buffer = Buffer.alloc(minimumBytes);
+
+	await tokenizer.peekBuffer(buffer, {position: options.position, length: 512, mayBeLess: true});
```

Comment:
`position: options.position` is the default, redundant.

Reply:
Sorry, I'm a bit confused about what you mean by this 😅 
`options.position` can be changed depending if you pass in an options object to `_checkSequence`... Am I missing something?

Reply:
Sorry I misread `position: tokenizer.position` is the default value. Better not to use absolute positions.

Reply:
Solved in https://github.com/sindresorhus/file-type/pull/323/commits/dc45850aa3aafe47eb41be288e46b54f78bb393b